avm/res/network/ddos-protection-plan/tests/e2e/defaults/main.test.bicep

```
@@ -20,6 +20,9 @@ param serviceShort string = 'ndppmin'
 @description('Optional. A token to inject into the name of each resource.')
 param namePrefix string = '#_namePrefix_#'
 
+#disable-next-line no-hardcoded-location // Due to quotas and capacity challenges, this region must be used in the AVM testing subscription
```

It's a very minor thing please bare with me, just in alignment with other implementations
```suggestion
// Due to quotas and capacity challenges, this region must be used in the AVM testing subscription
#disable-next-line no-hardcoded-location 
```